docs/INSTALL.md

```diff
@@ -0,0 +1,94 @@
+| [PhyloPhlAn](https://github.com/davidemms/OrthoFinder) | [1/25/2017](https://bitbucket.org/nsegata/phylophlan/commits/2c0e61ad820b8ff732837e98f7843afbb7ec1cda) | genome tree building | yes | yes | [23942190](https://www.ncbi.nlm.nih.gov/pubmed/23942190) |
+||
+| [BLAST+](https://blast.ncbi.nlm.nih.gov/Blast.cgi) | [2.5.0](ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.5.0/ncbi-blast-2.5.0+-src.tar.gz) | sequence similarity search | yes | yes | [2231712](https://www.ncbi.nlm.nih.gov/pubmed/2231712) |
+| [DIAMOND](https://ab.inf.uni-tuebingen.de/software/diamond) | [0.8.28](https://github.com/bbuchfink/diamond/archive/v0.8.28.tar.gz) | sequence similarity search | yes | yes | [25402007](https://www.ncbi.nlm.nih.gov/pubmed/25402007) |
+||
+| [MAFFT](http://mafft.cbrc.jp/alignment/software/) | [7.305](http://mafft.cbrc.jp/alignment/software/mafft-7.305-with-extensions-src.tgz)  | sequence alignment | yes | yes | [12136088](https://www.ncbi.nlm.nih.gov/pubmed/12136088) |
+| [MUSCLE](http://drive5.com/muscle/) | [3.8.31](http://www.drive5.com/muscle/downloads3.8.31/muscle3.8.31_i86linux64.tar.gz) | sequence alignment | yes | yes | [15034147](https://www.ncbi.nlm.nih.gov/pubmed/15034147) |
+| [KAlign](http://www.ebi.ac.uk/Tools/msa/kalign/) | [2.03](http://msa.sbc.su.se/downloads/kalign/current.tar.gz) | sequence alignment | no | yes | [16343337](https://www.ncbi.nlm.nih.gov/pubmed/16343337) |
+| [T-Coffee](http://www.tcoffee.org/) | [11.00.8cbe486](http://www.tcoffee.org/Packages/Stable/Latest/T-COFFEE_distribution_Version_11.00.8cbe486.tar.gz) | alignment combining | yes | yes | [10964570](https://www.ncbi.nlm.nih.gov/pubmed/10964570)
+||
+| [RAxML](http://sco.h-its.org/exelixis/web/software/raxml/index.html) | [8.2.9](https://github.com/stamatak/standard-RAxML/archive/v8.2.9.tar.gz) | tree building | yes | yes | [24451623](https://www.ncbi.nlm.nih.gov/pubmed/24451623) |
+| [PhyML](http://www.atgc-montpellier.fr/phyml/) | [3.2.20160530](https://github.com/stephaneguindon/phyml/archive/v3.2.20160530.tar.gz) | tree building | yes | yes | [20525638](https://www.ncbi.nlm.nih.gov/pubmed/20525638) |
+| [FastTree](http://www.microbesonline.org/fasttree/) | [2.1.9](http://www.microbesonline.org/fasttree/FastTree) | tree building | yes | yes | [20224823](https://www.ncbi.nlm.nih.gov/pubmed/20224823) |
+||
+| [MCL](http://micans.org/mcl/) | [14-137](http://micans.org/mcl/src/mcl-14-137.tar.gz) | clustering | no | yes | [22144159](https://www.ncbi.nlm.nih.gov/pubmed/22144159) |
+| [Prodigal](http://prodigal.ornl.gov/) | [2.6.3](https://github.com/hyattpd/Prodigal/archive/v2.6.3.tar.gz) | gene calling | yes | yes | [20211023](https://www.ncbi.nlm.nih.gov/pubmed/20211023) |
+
+#### Conda install
+
+The following programs can be installed directly from conda:
+
+Available in [bioconda](https://anaconda.org/bioconda):
```

Comment:
I sat with @tkosciol and explained. The sad fact is that there is no way to automate the entire installation process. Some apps, such as **usearch**, requires that the user to go to their website, fill in name and email address and receive a licensed copy in their inbox. Therefore I suggest that we describe the installation procedures in detail, instead of / in addition to having an automated script.